kdoctor/src/commonMain/kotlin/org/jetbrains/kotlin/doctor/Arguments.kt

```diff
@@ -20,6 +20,13 @@ class Arguments(args: Array<String>) {
         description = "print extended information"
     ).default(false)
 
+    val isFull: Boolean by parser.option(
+        ArgType.Boolean,
+        shortName = "f",
```

Comment:
For me`-f` means only `--force`. Maybe it'd be better to drop the short name? Or think on another e.g. `--all`?